### DISCUSSION TITLE
I want to add a button on my website. When clicked, it should open the GrapesJS editor, and my template should also be loaded into GrapesJS so that I can edit it.

### DISCUSSION THREAD

**Discussion by ZaheerNiazi007** [GrapesJS/grapesjs#5873] [Ideas]
![Screenshot 2024-05-08 122455](https://github.com/GrapesJS/grapesjs/assets/72734684/97dcbfcb-489a-405a-9276-0e73e00738dd)
**When click on Open Editor this template will open in GrapesJS**

**Comment by artf** [comment]
Please be careful to avoid duplicates https://github.com/GrapesJS/grapesjs/discussions/5874
